test_project/test_app/tests/test_user.py

```diff
@@ -14,6 +14,13 @@
 User = get_user_model()
 
 
+def encode_string(str):
```

Comment:
Can we put this in a `utils.py` file? I can see this function being useful elsewhere.